.travis.yml

```diff
@@ -2,14 +2,22 @@ language: python
 python: 3.6
 cache: pip
 env:
-  - REPO_NAME="${PWD##*/}"
+  global:
+    - REPO_NAME="${PWD##*/}"
+    - PIPENV_IGNORE_VIRTUALENVS=1
+    - SAM_CLI_TELEMETRY=0
+    - AWS_DEFAULT_REGION="us-east-1"
```

Comment:
these AWS env vars are not needed.  AWS configs are setup in `before_script` section.  it should look like our standard lambda template: https://github.com/Sage-Bionetworks-IT/lambda-template/blob/master/.travis.yml